dockers/docker-dhcp-relay/start.sh

```diff
@@ -5,7 +5,7 @@ if [ "${RUNTIME_OWNER}" == "" ]; then
 fi
 
 CTR_SCRIPT="/usr/share/sonic/scripts/container_startup.py"
-if test -f ${CTR_SCRIPT}
+if [ $RUNTIME_OWNER = "kube" ] && test -f ${CTR_SCRIPT}
```

Comment:
>test [](http://example.com/codeflow?start=34&length=4)

I am confused:
If $RUNTIME_OWNER=="kube" and CTR_SCRIPT does not exist, you run the script?

Reply:
This code means if $RUNTIME_OWNER=="kube" and CTR_SCRIPT exists, than I run the script